tensorflow/python/ops/math_ops.py

```diff
@@ -646,6 +646,35 @@ def imag(input, name=None):
     return gen_math_ops.imag(input, Tout=input.dtype.real_dtype, name=name)
 
 
+def arg(input, name=None):
```

Comment:
I don't believe the wrapper is necessary, as the default wrapper will have the exact same functionality. Are you just doing this to provide a python example and to use a name_scope wrapper?